hydrolib/core/dflowfm/extold/__init__.py

```diff
@@ -0,0 +1,10 @@
+from ..extold.models import ExtForcing, ExtOldModel
+
+__all__ = [
+    "ExtForcing",
+    "ExtOldModel",
+    "Quantity",
```

Comment:
To fix the bugs mentioned by Sonar, I think you have to update the import statement above:

`from .models import ExtForcing, ExtOldModel, Quantity, FileType, Method, Operand`

Reply:
Done

Reply:
Let's also add the new `TracerQuantity`

Reply:
Done